tests/SConscript

```diff
@@ -10,6 +10,7 @@ afwdataDir = env.ProductDir("afwdata")
 if afwdataDir:
     env["ENV"]["AFWDATA_DIR"] = afwdataDir
 
-scripts.BasicSConscript.tests(noBuildList=[name + '.cc' for name in pybind11_test_modules])
+scripts.BasicSConscript.tests(noBuildList=[name + '.cc' for name in pybind11_test_modules],
+                              ignoreList=[name + '.py' for name in pybind11_test_modules])
```

Comment:
This is needed to stop SCons from thinking that the pybind11 .py files are things that should be tested. This is important when we are in a mode where pytest runs each test file separately because pytest exits with bad status if no tests are found. See my comment above replying to @kfindeisen. With these changes `pytest` without arguments would work but `sconsUtils` is not yet set up to do that (that's a change we can make later once the basics are done).